### DISCUSSION TITLE
Specification for integrating external services in the UCI chat flow

### DISCUSSION THREAD

**Discussion by kameshbhr** [samagra-comms/community#24] [New Features]
There is a requirement to integrate external services in the UCI Whatsapp Chat flow. The following slides describe the requirement, solution flow and possible high level tech approach: https://docs.google.com/presentation/d/1ZrrV8579Ncto4-VRctGP84ymIpjUNDV-VOUwfIf1_pw/edit?usp=sharing 

@rayulu @ChakshuGautam Can you please review this. I need your help with the following:
1. Reviewing the tech approach
2. Defining spec for registering and using an external service

Will schedule a call to go through the same.

**Comment by vrayulu** [comment]
@ChakshuGautam - please share the available documentation for current UCI - architecture, capabilities and specs (if any).

**Comment by ChakshuGautam** [comment]
Hey, @kameshbhr let's discuss this during the call scheduled for tomorrow. The current docs on the [microsite](https://uci.sunbird.org/) are updated, so please take a look there before tomorrow's call. Thanks.
